dts/bindings/sensor/ti,ina230.yaml

```diff
@@ -1,49 +1,29 @@
 #
 # Copyright 2021 The Chromium OS Authors
+# Copyright 2021 Grinn
 #
 # SPDX-License-Identifier: Apache-2.0
 #
 
-
 description: |
     TI INA230 and INA231 Bidirectional Current and Power Monitor.
-    The <include/dt-bindings/sensor/ina23x.h> file should be included
+    The <include/dt-bindings/sensor/ina230.h> file should be included
     in the DeviceTree and it provides Macros that can be used for
     initializing the Configuration register.
 
-compatible: "ti,ina23x"
+compatible: "ti,ina230"
 
-include: i2c-device.yaml
+include: ti,ina23x-common.yaml
 
 properties:
-    config:
-      type: int
-      required: true
-      description: Configuration register
-
-    current_lsb:
-      type: int
-      required: true
-      description: |
-        Value of Current LSB in milliamps. When set to 1mA,
-        Current is read in A, Bus Voltage in V, Shunt
-        Voltage in V, and Power in mW. Any other value
-        results in Current, Voltage, and Power registers
-        being read in counts.
-
-    rshunt:
-      type: int
-      required: true
-      description: Shunt resistor value in milliohms
-
     mask:
       type: int
       required: false
       default: 0
       # default all alert sources to disabled
       description: Mask register, default matches the power-on reset value
 
-    alert_limit:
+    alert-limit:
```

Comment:
It is worth noting in the commit message, that various function doxygen documentation has been updated and also some device-tree node names now use `-` instead of `_` in order to follow convention.

Reply:
Fixed.